### PR TITLE
[12.0][FIX] web_action_conditionable: Adapt to new api

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Forward port of https://github.com/OCA/web/pull/1343/

Current code is not working and it is raising errors.

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
cc @ForgeFlow

**Review by AdriaGForgeFlow — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
image
fixes this?

**Comment by AaronHForgeFlow** [comment]
@alexey-pelykh with the current code, if you define a tree view like this:

`<tree delete="state=='draft'">`

you would get an error (https://github.com/OCA/web/pull/1395#pullrequestreview-400031278 ) and no chance to load the view.

This fix will allow to define that kind of tree views in one2many fields.

Regards

**Comment by alexey-pelykh** [comment]
@AaronHForgeFlow it seems that right now in v12 this module simply does not load :(

**Comment by AaronHForgeFlow** [comment]
> @AaronHForgeFlow it seems that right now in v12 this module simply does not load :(

I will check again. Thanks.

**Comment by AaronHForgeFlow** [comment]
@alexey-pelykh  I just tested and yes, this module does not load now in v12. Just tested and with this fix it work correctly. Could you check?

File: web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api [hunk 1]
```diff
@@ -0,0 +1,435 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (alexey-pelykh) [inline web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api]: I doubt this file is needed

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api (hunk 1 above)
Comment (AaronHForgeFlow) [reply web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api]: I am not sure, maybe @Tardo can shed some light here

Comment (Tardo) [reply web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api]: This looks like a temporary file... :fire:

Comment (AaronHForgeFlow) [reply web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api]: thank you

**Review by marcelsavegnago — APPROVED** [review]
LGMT

**Comment by hoangtrann** [comment]
It seem that changes proposed by this PR are merged into branch `11.0`, `13.0` but aren't getting merged into `12.0`.

Could we take a look on this again, please?

**Review by Tardo — APPROVED** [review]

**Pull request merged**